pkgs/tools/networking/eternal-terminal/default.nix

```diff
@@ -2,13 +2,13 @@
 
 stdenv.mkDerivation rec {
   name = "eternal-terminal-${version}";
-  version = "4.1.2";
+  version = "5.1.0";
 
   src = fetchFromGitHub {
     owner = "MisterTea";
     repo = "EternalTCP";
     rev = "refs/tags/et-v${version}";
```

Comment:
Weird that they use explicit tag, even if there are no potentially conflicting branches.

Reply:
I can't remember the particular reason I do it. Was it perhaps that `fetchgit` doesn't work without `refs/tags/`? Maybe it's a habit...